iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/plugin/KubernetesParserTest.java

```diff
@@ -159,4 +160,37 @@ void shouldFallbackToFilenameInCaseOfUnresolvedChartDirectory() throws URISyntax
       assertThat(argumentCaptor.getValue()).isEqualTo("foo.yaml");
     }
   }
+
+  @Test
+  void shouldRemoveEmptyLinesAfterEvaluation() throws IOException, URISyntaxException {
+    try (var ignored = Mockito.mockStatic(HelmFilesystemUtils.class)) {
+      var valuesFile = mock(InputFile.class);
+      when(valuesFile.filename()).thenReturn("values.yaml");
+      when(valuesFile.contents()).thenReturn("foo: bar");
+      when(sensorContext.fileSystem().inputFile(any())).thenReturn(valuesFile);
+      String evaluatedSource = code("apiVersion: apps/v1 #1",
+        "kind: StatefulSet #2",
+        "metadata: #3",
+        "  name: helm-chart-sonarqube-dce-search #4",
+        "spec: #5",
+        "  livenessProbe: #6",
+        "    exec: #7",
+        "      command: #8",
+        "        - sh #9",
+        "        - -c #10",
+        "        #14",
+        "        - | #15",
```

Comment:
I'm surprised that this line is okay for snakeyaml (it even doesn't add `#15` into the node), but the previous line crashes it